tutorials/roofit/rf711_lagrangianmorph.py

```diff
@@ -0,0 +1,178 @@
+##
+## \macro_code
+##
+## \date January 2022
+## \authors Rahul Balasubramanian
+
+import ROOT
+
+ROOT.gStyle.SetOptStat(0)
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+ROOT.gROOT.SetBatch(True)
+
+# Create functions
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+observablename = "pTV"
+
+# Setup observable that is to be morphed
+obsvar = ROOT.RooRealVar(observablename, "observable of pTV", 10, 600)
+
+# Setup two couplings that enters the morphing function
+# kSM -> SM coupling set to constant (1)
+# cHq3 -> EFT parameter with NewPhysics attribute set to true
+kSM = ROOT.RooRealVar("kSM", "sm modifier", 1.0)
+cHq3 = ROOT.RooRealVar("cHq3", "EFT modifier", 0.0, 1.0)
+cHq3.setAttribute("NewPhysics", True)
+couplings = ROOT.RooArgSet()
+
+# Inputs to setup config
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+infilename = "input_histos_rf_lagrangianmorph.root"
+par = "cHq3"
+samplelist = ["SM_NPsq0", "cHq3_NPsq1", "cHq3_NPsq2"]
+
+# Set Config
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+config = ROOT.RooLagrangianMorphFunc.Config()
+config.fileName = infilename
+config.observableName = observablename
+config.folderNames = samplelist
+config.couplings.add(cHq3)
+config.couplings.add(kSM)
+
+
+# Create morphing function
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+morphfunc = ROOT.RooLagrangianMorphFunc("morphfunc", "morphed dist. of pTV", config)
+
+# Get morphed distribution at cHq3 = 0.01, 0.5
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+morphfunc.setParameter("cHq3", 0.01)
+morph_hist_0p01 = morphfunc.createTH1("morph_cHq3=0.01")
+morphfunc.setParameter("cHq3", 0.25)
+morph_hist_0p25 = morphfunc.createTH1("morph_cHq3=0.25")
+morphfunc.setParameter("cHq3", 0.5)
+morph_hist_0p5 = morphfunc.createTH1("morph_cHq3=0.5")
+morph_datahist_0p01 = ROOT.RooDataHist("morph_dh_cHq3=0.01", "", ROOT.RooArgList(obsvar), morph_hist_0p01)
+morph_datahist_0p25 = ROOT.RooDataHist("morph_dh_cHq3=0.25", "", ROOT.RooArgList(obsvar), morph_hist_0p25)
+morph_datahist_0p5 = ROOT.RooDataHist("morph_dh_cHq3=0.5", "", ROOT.RooArgList(obsvar), morph_hist_0p5)
+
+# Extract input templates for plotting
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+input_hists = {sample: ROOT.TFile.Open(infilename).Get(sample).FindObject(observablename) for sample in samplelist}
+input_datahists = {
+    sample: ROOT.RooDataHist("dh_" + sample, "dh_" + sample, ROOT.RooArgList(obsvar), input_hists[sample])
+    for sample in samplelist
+}
+
+# Plot input templates
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+frame0 = obsvar.frame(ROOT.RooFit.Title("Input templates for p_{T}^{V}"))
```

Comment:
```suggestion
frame0 = obsvar.frame(Title="Input templates for p_{T}^{V}")
```
Use new Pythonizations.